Extension/src/common.ts

```diff
@@ -972,9 +987,9 @@ export function extractCompilerPathAndArgs(inputCompilerPath?: string, inputComp
             // Input has no quotes but can have a compiler path with spaces and args.
             // Go from right to left checking if a valid path is to the left of a space.
             let spaceStart: number = compilerPath.lastIndexOf(" ");
-            if (spaceStart !== -1 && (!isWindows || !compilerPath.endsWith("cl.exe")) && !checkFileExistsSync(compilerPath)) {
+            if (spaceStart !== -1 && (!isWindows || !compilerPath.toLowerCase().endsWith("cl.exe")) && !checkFileExistsSync(compilerPath)) {
```

Comment:
What is `compilerPath.toLowerCase().endsWith("cl.exe")` accomplishing on line 990?  If `compilerPath` ended with a filename of "cl.exe", it would have been caught on line 974.  So, this would only detect clang-cl.exe, which I don't think we should be do anything for here, anymore.  I think this IF statement should be reduced to just `if (spaceStart !== -1 && !checkFileExistsSync(compilerPath))`.  Also, Nit: Could you use a new var to avoid calling toLowerCase() multiple times on `compilerPath`, on line 974?  :)